PlannTer/UI/Inputs/DropdownPicker.swift

```diff
@@ -0,0 +1,88 @@
+import SwiftUI
+
+struct DropdownPicker: View {
```

Comment:
they need to get automatically pulled back up if user taps anywhere outside of the picker.